MSAL/src/instance/MSALAuthority.m

```diff
@@ -157,7 +157,9 @@ + (void)resolveEndpointsForAuthority:(NSURL *)unvalidatedAuthority
         tenant = firstPathComponent;
     }
     
-    MSALAuthority *authorityInCache = [MSALAuthority authorityFromCache:updatedAuthority userPrincipalName:userPrincipalName];
+    MSALAuthority *authorityInCache = [MSALAuthority authorityFromCache:updatedAuthority
+                                                          authorityType:authorityType
+                                                      userPrincipalName:userPrincipalName];
     
```

Comment:
(just in case, I refer to the resolveEndpointsForAuthority:userPrincipalName:validate:context:completionBlock method itself)